src/Microsoft.Dnx.Tooling/Program.cs

```diff
@@ -80,7 +80,28 @@ public int Main(string[] args)
             WrapConsoleCommand.Register(app, reportsFactory);
             FeedsConsoleCommand.Register(app, reportsFactory);
 
-            return app.Execute(args);
+            try
+            {
+                return app.Execute(args);
+            }
+            catch (CommandParsingException ex)
+            {
+                AnsiConsole.GetError(_runtimeEnv.OperatingSystem == "Windows").WriteLine(ex.Message.Red().Bold());
```

Comment:
Cool, just wanted to check :). Could you use a named parameter here to clarify the purpose of the parameter? Something like:

```csharp
AnsiConsole.GetError(whatever_the_parameter_name_is: _runtimeEnv.OperatingSystem == "Windows").WriteLine(ex.Message.Red().Bold());
```